readme.md

```diff
@@ -1,3 +1,17 @@
+# ⚠️ Notice: this repository has been deprecated ⚠️
+
+This .NET 4 sample has been replaced with the new .NET Core Smartsheet C# SDK and can be accessed [here](https://github.com/smartsheet/smartsheet-csharp-sdk/tree/mainline/sdk-csharp-sample-net80)
+
+## See also
+- https://github.com/smartsheet/smartsheet-csharp-sdk
+- http://smartsheet-platform.github.io/api-docs/
```

Comment:
Might want to update api docs to point to actual new docs. https://smartsheet.redoc.ly/

Reply:
done